pennylane_lightning/lightning_qubit.py

```diff
@@ -576,6 +576,33 @@ def probability(self, wires=None, shot_range=None, bin_size=None):
 
         return M.probs(device_wires)
 
+    def generate_samples(self):
+        """Generate samples
+
+        Returns:
+            Array of samples in binary
```

Comment:
Is this correct to assume that the return value is as follows?
```suggestion
             array[complex]: array of samples in the shape ``(dev.shots, dev.num_wires)``
```

Reply:
Changed to array[int] because the pennylane documentation has it wrong. Thank you for spotting this!